salt/utils/network.py

```diff
@@ -1958,3 +1959,45 @@ def parse_host_port(host_port):
             raise ValueError('bad hostname: "{}"'.format(host))
 
     return host, port
+
+
+def _filter_by_networks(values, networks):
+    '''
+    Filter address list against a list of networks.
+    '''
+    filtered_addresses = []
+
+    for value in values:
+        address = ipaddress.ip_address(value)
+        for network in networks:
+            if address in network:
+                filtered_addresses.append(value)
+
+    return filtered_addresses
+
+
+@jinja_filter('filter_by_networks')
+def filter_by_networks(values, filters=None):
```

Comment:
Nit, maybe is better to rename `values` as `addresses`, and `filters` as `networks`

Reply:
I agree, I'll update the names